src/Microsoft.ML.Core/Data/ColumnType.cs

```diff
@@ -496,7 +496,7 @@ public static NumberType Float
             get { return R4; }
         }
 
-        public new static NumberType FromKind(DataKind kind)
+        new public static NumberType FromKind(DataKind kind)
```

Comment:
❕  The previous ordering is definitely preferred by a majority of projects. Would encourage correcting this analysis.

:memo: Accessibility ordering analysis is available through a couple of maintained alternatives, which should help avoid analysis bugs long-term.

Reply:
Are you sure? All samples I see with this tend to prefer `new` first...

Reply:
All right sounds fine. Replaced with SC rules [SA1205](https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1205.md), [SA1206](https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1206.md), [SA1207](https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1207.md), [SA1212](https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1212.md), [SA1400](https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1400.md).